projects/plugins/social/src/class-rest-controller.php

```diff
@@ -62,15 +62,58 @@ public function require_admin_privilege_callback() {
 		);
 	}
 
+	/**
+	 * A list of all settings that can be retrieved and updated..
+	 *
+	 * Items for each setting:
+	 * - key: Key to be posted in the request body.
+	 * - get_callback: Function to get the current value of the setting.
+	 * - update_callback: Function to call to update the setting.
+	 * - validation_callback: Gets the return value of the update_callback.
+	 *     Return a boolean to indicate if the setting updated successfully.
+	 *
+	 * @return array
+	 */
+	public function get_available_settings() {
+		return array(
+			array(
+				'key'                 => 'publicize_active',
+				'get_callback'        => function () {
```

Comment:
I notice this is repeated in [the main Jetpack_Social class](https://github.com/Automattic/jetpack/pull/26213/files#diff-d2a85dafd23b905630dba0fd25897b63c873bfea0d0e83b9a3ce237f1dbde996R166). Perhaps we should define it as a named function somewhere.

Reply:
Done in [390d332](https://github.com/Automattic/jetpack/pull/26213/commits/390d3321d5f8e2970b8c2295675fa341f64def62).